src/main/java/testUI/elements/TestUI.java

```diff
@@ -115,26 +115,26 @@ protected String getAccesibilityId(String accesibilityIdiOS, String accesibility
     protected List getMobileElementList(String locator) {
```

Comment:
Update this function signature to get rid of unnecessary casting, also update final `return` in `default` case (can't comment on unchanged lines 🥲 )
```suggestion
    protected List<WebElement> getMobileElementList(String locator) {
```